assets/i18n/en.json

```diff
@@ -93,17 +93,28 @@
     "I18N_DASHBOARD_EXPLORATIONS_SORT_BY_TOTAL_PLAYS": "Total Plays",
     "I18N_DASHBOARD_EXPLORATIONS_SORT_BY_UNRESOLVED_ANSWERS": "Unresolved Answers",
     "I18N_DASHBOARD_OPEN_FEEDBACK": "Open Feedback",
+    "I18N_DASHBOARD_SKILLS": "Skills",
```

Comment:
Ah -- there may have been a misunderstanding. I think we should remove strings we aren't planning to translate, so that we do not unnecessarily burden the translatewiki folks. Could we do that?

Reply:
Oh, ok, except the title in the navbar, removed everything lese from translations.